conda_forge_tick/make_graph.py

```diff
@@ -11,119 +11,110 @@
 logger = logging.getLogger("conda_forge_tick.make_graph")
 
 
-def get_attrs(name, i, bad):
+def get_attrs(name, i):
     sub_graph = {
-        'time': time.time(),
-        'feedstock_name': name,
+        "time": time.time(),
+        "feedstock_name": name,
         # All feedstocks start out as good
-        'bad': False
+        "bad": False,
     }
 
     logger.info((i, name))
-    r = requests.get('https://raw.githubusercontent.com/'
-                     'conda-forge/{}-feedstock/master/recipe/'
-                     'meta.yaml'.format(name))
+    r = requests.get(
+        "https://raw.githubusercontent.com/"
+        "conda-forge/{}-feedstock/master/recipe/"
+        "meta.yaml".format(name)
+    )
     if r.status_code != 200:
-        logger.warn('Something odd happened when fetching recipe '
-                    '{}: {}'.format(name, r.status_code))
-        bad.append(name)
-        sub_graph['bad'] = r.status_code
+        logger.warn(
+            "Something odd happened when fetching recipe "
+            "{}: {}".format(name, r.status_code)
+        )
+        sub_graph["bad"] = r.status_code
```

Comment:
I liked what you did with the update bad, would it be possible to do something similar here? eg. `'make_graph: {}'.format(r.status_code)`